doc/sidewalk_known_issues.rst

```diff
@@ -8,21 +8,23 @@ See the list of known issues that are valid for the current state of development
 v1.14.5-dev1
 ------------
 
-* KRKNWK-17374: Sporadic Zephyr fatal error after disconnecting on FSK
-    After disconnecting on FSK, Zephyr fatal error occurrs due to assertion in semaphore module.
-    The error reproduces rarely - once per few days.
-    In the release mode the device resets automatically, however, in the debug mode it need to be reset manually.
-    Currently, this issue occurs for Sidewalk 1.14 libraries and it will be fixed in the version 1.16.
+* KRKNWK-17860: QSPI assert when trying to perform dfu in sidewalk application mode
+    The DFU should be performed only in dfu mode. However the dfu bluetooth service is available in sidewalk mode, and using it lead to assertion failure witch means a zephyr fatal error.
 
-* KRKNWK-17321: The device fails after multiple restarts on the nRF5340 DK.
-   The issue happens due to failing certification test.
+* KRKNWK-17800: After network connection is back end device cannot find route to host.
+    After losing connection to Sidewalk servers, sensor demo app over ble stuck with error -38 (SID_ERROR_NO_ROUTE_AVAILABLE). Device need to be reset manually.
 
-   **Affected platforms:** nRF5340
+* KRKNWK-17750: Sidewalk errors on multiple messages send in a short time period.
+    When sending multiple sidewalk messages in short period of time internal queues may be full and cause misleading errors messages like -12 (SID_ERROR_INCOMPATIBLE_PARAMS). The message should be resend after some amount of time needed for protocol to empty the queues.
 
-* KRKNWK-17114: Bootloader configuration is not optimized on the nRF5340 DK.
-   Due to large bootloader partition size (48 kB), the DFU always requires an external flash.
+* KRKNWK-17244:  CMake warnings during Sidewalk samples build.
+    "No SOURCES given to Zephyr library" CMake warnings present in build log of Sidewalk application.
```

Comment:
```suggestion
* KRKNWK-17244:  CMake warnings when building the Sidewalk samples
    ``No SOURCES given to Zephyr library`` CMake warnings shows in the build log of Sidewalk application.
```
Is there anything additional that can be added to this? E.g., what should user do when they see it? Will it create any other issues?

Reply:
Added an extra sentence